spec/datadog/tracing/contrib/rails/support/base.rb

```diff
@@ -6,9 +6,6 @@
   require 'datadog/tracing/contrib/sidekiq/server_tracer'
 end
 
-require 'lograge' if ENV['USE_LOGRAGE'] == true
-require 'rails_semantic_logger' if ENV['USE_SEMANTIC_LOGGER'] == true
```

Comment:
I'm curious -- why are these requires no longer needed?

I'm asking because I remember that in the past a few things "auto-probed" what was enabled, and thus it looked like the tests were actually testing what we wanted but since something wasn't loaded it wasn't actually being tested.